.github/workflows/stale_community_prs.yml

```diff
@@ -0,0 +1,58 @@
+name: 'Check stale community PRs.'
+on: workflow_dispatch
+      
+jobs:
+  stale-implementation-stage:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@3cc123766321e9f15a6676375c154ccffb12a358
+        with:
+          debug-only: true
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          stale-pr-label: closing-soon
+          stale-pr-message: 'This pull request is in the implementation stage. It has become stale because it has been 20 days with no activity. Please comment with an update or this will be closed in 10 days.'
+          close-pr-message: 'This pull request was closed because it has been stalled for 10 days with no activity.'
+          days-before-stale: 20
+          days-before-close: 10
+          days-before-issue-close: -1
+          days-before-issue-stale: -1
+          only-pr-labels: community,implementation,response-requested
+          exempt-pr-labels: responded
```

Comment:
Maybe let's see if we need the `exempt-pr-labels` if `only-pr-labels` are going to be AND based. Or if we want to keep it here maybe add a note on why we have it.